service/docs/source/install.rst

```diff
@@ -1,115 +1,86 @@
-
-
-
-Use these links to install the GEOPM Service packages on your system.  The
-links contain instructions and examples for each of the supported operating
-systems.
-
-Note that the development branch always provides a version which is at
-least as recent as the release branch.  For this reason, updates will
-always come from the development branch download repository if both
-the development repository and the release repository are added to
-your package configuration.
-
-
-Un-packaged Features
---------------------
-
-The packages described above do not provide the
-:doc:`GEOPM HPC Runtime <runtime>` features (e.g.
-:doc:`geopmlaunch(1) <geopmlaunch.1>`, :doc:`geopm_prof(3) <geopm_prof.3>`,
-:doc:`geopm_report(7) <geopm_report.7>`, ``libgeopm.so``, etc.).
-The GEOPM HPC Runtime must be built against a particular implementation of the
-Message Passing Interface (MPI).  Typically the implementation of MPI is
-specific to the system where the HPC application is being run.  An exception
-to this rule is the OpenHPC distribution.  GEOPM version 1 has been packaged
-with `OpenHPC releases <http://openhpc.community/downloads/>`_, and we hope to distribute version 2 with OpenHPC in the future.
-For users interested in GEOPM version 2, a source build is required.
-
-Support for NVIDIA GPUs with NVML or DCGM is also not provided by
-installing the packages described above.  Support for NVIDIA GPUs
-relies on libraries that are not bundled with the OpenSUSE OBS build
-system where the GEOPM packages are built and distributed.  A source
-build of the GEOPM Service is required to enable use of NVML, or DCGM.
-This build process must be done in an environment where the dependency
-libraries and headers are present.
-
-In order to access these GEOPM features which are not packaged, a user should
-build GEOPM from source.  The best recommendation for building GEOPM from
-source is to follow the :ref:`developer build process <devel:developer build process>`
-posted in the :doc:`developer guide <devel>`.  Note that it may be of interest
-to git checkout a git tag (e.g. ``v2.0.0``) to create a build based on a
-particular release.
+The ``release-v2.0`` branch logs v2.0 releases, including any potential
+hotfixes for v2.0.0. It captures the latest stable GEOPM release. The ``dev``
+branch, however, presents the most updated stable development. Builds from
+these two branches have the following download repository pages:
+
+- Release Packages Installation (``release-v2.0`` branch)
+   - Refer links provided
+
+- Development Packages Installation (``dev`` branch)
+   - Refer links provided
+
+Use the provided links to install GEOPM Service packages on your system. The
+links will also guide you through instructions for each supported operating
+system. If you add both the development repository and the release repository
+to your package configuration, you will always receive updates from the
+development branch.
```

Comment:
actually, I'm not sure this is true once a hot fix is created.  Once there is a hotfix, that version will be prefered by the package manager over the dev packages, e.g.: 2.0.1 > 2.0.0+dev999gabcdef1

Reply:
Advise the user add one repo or the other, not both.